src/data/local/github/tracks/track-2/steps/step-3.md

```diff
@@ -2,7 +2,7 @@
 
 - Open a Pull Request of your `feature/setup` branch
 
-If you don't remember how to open a Pull Request, check out the references in the previous issues.
+If you don't remember how to open a Pull Request, check out the references in the previous issues. Your PR should point to `develop` branch, to be merged later.
```

Comment:
Talvez `Your PR should be the `develop` branch as the base, to be merged into later.` ?